cmd/apps/istio_app.go

```diff
@@ -28,119 +30,136 @@ func MakeInstallIstio() *cobra.Command {
 		Example:      `  arkade install istio --loadbalancer`,
 		SilenceUsage: true,
 	}
-	istio.Flags().Bool("update-repo", true, "Update the helm repo")
-	istio.Flags().String("namespace", "istio-system", "Namespace for the app")
-	istio.Flags().Bool("init", true, "Run the Istio init to add CRDs etc")
+	istio.Flags().StringP("version", "v", istioVer, "Specify a version of Istio")
+	istio.Flags().StringP("manifests", "d", "https://github.com/istio/istio/releases/download/1.9.0/istio-1.9.0-linux-amd64.tar.gz", "Specify a path to a directory of charts and profiles")
```

Comment:
Doesn't Istio have ARM64 support now? https://istio.io/latest/docs/setup/install/istioctl/

How would we switch that for the user, assuming they are using a Graviton or RPi with Ubuntu?

Reply:
They do support images for these.

Reply:
The way I read it, you are always bringing in `manifests` if it has a non zero length, and it always does because it's hard-coded.

Have you thought of using `.Changed("manifests")` to detect whether it's been overridden by the user? That or, remove the default.

Reply:
We won't need it now, but in future will use it.